recipes/environ-config/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "environ-config" %}
+{% set version = "21.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/environ-config-{{ version }}.tar.gz
+  sha256: d82fa2f760ac031fb9f434be56a0a83e98dbeba8151cdd96feaf86cd17417dd9
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - attrs >=17.4.0
+    - configparser  # [py<30]
```

Comment:
Unless you absoutely need Python 2 support yourself, conda-forge doesn't really support Python 2.7, so it would be best to keep this `noarch` and remove the stuff needed to support Python 2.